bot/exts/utilities/reddit.py

```diff
@@ -43,8 +42,8 @@ def __init__(self, bot: Bot):
     async def cog_unload(self) -> None:
         """Stop the loop task and revoke the access token when the cog is unloaded."""
         self.auto_poster_loop.cancel()
-        if self.access_token and self.access_token.expires_at > datetime.utcnow():
-            asyncio.create_task(self.revoke_access_token())
+        if self.access_token and self.access_token.expires_at > datetime.now(tz=UTC):
+            _ = asyncio.create_task(self.revoke_access_token())
```

Comment:
This wouldn't fix the potential issue with tasks being discarded. probably not really an issue, I wonder if we could just await the coroutine here though?

Reply:
Yea, we can do that. Was likely a hold over from before cog_unload was async https://github.com/python-discord/sir-lancebot/pull/1270/commits/3887b52eeb6443c9b84cb537ebad4b0526efd9f2